src/content/nft/index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Non-fungible tokens (NFT)
+description: An overview of NFTs on Ethereum
+lang: en
+template: use-cases
+emoji: ":frame_with_picture:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/infrastructure_transparent.png
+summaryPoints:
+  [
+    "A way to represent anything unique as an Ethereum-based asset.",
+    "NFTs are giving more power to content creators than ever before.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans. But digital art is only one way to use NFTs. Really they can be used to represent ownership of any unique asset, like a deed for an item in the digital or physical realm.
+
+_If Andy Warhol had been born in the late 90s, he probably would have minted Campbell's Soup as an NFT. It's only a matter of time before Kanye puts a run of Yeezys on Ethereum. And one day owning your car might be managed with NFTs._
+
+<InfoBanner isWarning>NOTE: we don't currently explain the problems NFTs solve. On defi we talk about the problems with the tradfi system and on daos we talk about tthe difficulties of trusting strangers. The internet today isn't set up for ownership, but how can we easily explain this?</InfoBanner>
```

Comment:
What problems are NFTs solving? Here's a few that stick out to me:

1. Digital scarcity
    * No provable uniqueness in the current internet… music/video copy is the same file as the original
2. Digital ownership
    * Hard to verify ownership of something. Typically requires some "credible" central party assuring who owns what.
3. Interoperability
    * In current internet world, there is none. e.g. all concert ticket companies… Eventbrite, Ticketmaster… none of them are interoperable with our wallets. Need to build their own apps & marketplaces & partnerships with venues 
    * NFTs can "plug in" to existing infrastructure (e.g. DEXs to trade, DeFi protocols to borrow money using NFTs as collateral). Empowers creators - no longer reliant on services for this infrastructure / distribution & restrained by their terms of use / geographic limitations

Reply:
Along w/ #3, because there's interoperability, there's global liquidity when you want to exit 😄